packages/application-config/src/formatters.ts

```diff
@@ -118,6 +122,15 @@ function entryPointUriPathToResourceAccesses<
   };
 }
 
+function resolveCustomViewResourceAccesses<PermissionGroupName extends string>(
```

Comment:
When writing the docs for Custom Views I noticed if a developer needs to calculate the permissions he would need to use something like this:
```js
export const PERMISSIONS = entryPointUriPathToPermissionKeys(CUSTOM_VIEW_HOST_ENTRY_POINT_URI_PATH);
```

It seems confusing to me as the concept of `entry point` does not exist for Custom Views, so I decided to propose the inclusion of this new helper target for Custom Views where they can get both the basic resource accesses and also the one for the additional groups.

Reply:
@emmenko please let me knot what you think about this 🙏

Reply:
Thanks for the suggestion.

I updated this part here: 96cc64e73e425db269f70c601169871eed7eff6f